cmd/csi-attacher/main.go

```diff
@@ -190,7 +189,7 @@ func main() {
 		if csiFactory != nil {
 			csiFactory.Start(stopCh)
 		}
-		ctrl.Run(threads, stopCh)
+		ctrl.Run(*workerThreads, stopCh)
```

Comment:
Please check that the value is `> 0`

Reply:
Fixed. Use Uint flag and check the value before used